src/components/dialogs/connectivity-edition.js

```diff
@@ -138,7 +138,7 @@ export const useConnectivityValue = ({
         label: 'ConnectionDirection',
         validation: { isFieldRequired: false },
         inputForm: inputForm,
-        formProps: filledTextField,
+        formProps: outlinedTextField,
```

Comment:
Are you sure you have to give explicitly this property ?
I see in the documentation that 'outlined' is the default variant for a TextField

Reply:
yes you are right, ill delete this prop in this case